engine/src/main/java/org/terasology/rendering/dag/nodes/SimpleBlendMaterialsNode.java

```diff
@@ -78,7 +73,7 @@ public void process() {
      * be reversed, not eliminated, by re-enabling writing to the Depth Buffer.
      */
     private void preRenderSetupSimpleBlendMaterials() {
-        frameBuffersManager.getFBO(SCENE_OPAQUE_FBO).setRenderBufferMask(true, true, true);
+        dynamicFBM.getFBO(DefaultDynamicFBOs.ReadOnlyGBuffer.getResourceUrn()).setRenderBufferMask(true, true, true);
```

Comment:
You should be able to do a simple: 

`DefaultDynamicFBOs.ReadOnlyGBuffer.setRenderBufferMask(true, true, true);`

no?